.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Description of the change
+<!--- Describe your changes in detail. -->
+
+
+## Reference
+<!--- Link to the reference implementation, research paper, and GitHub issue. -->
+
+## Demo Notebook
+<!-- If the change is related to a new model, provide an end-to-end working demonstration of the model usage, training, and numerics verification. -->
```

Comment:
If adding any new API, attach a colab showing the high-level usage. If adding a model, please also include numerics verification against a reference implementation.